packages/cli/src/commands/doctor/healthchecks/androidSDK.ts

```diff
@@ -61,13 +82,88 @@ export default {
       needsToBeFixed: !isRequiredVersionInstalled,
     };
   },
-  runAutomaticFix: async ({loader, environmentInfo}) => {
-    const version = environmentInfo.SDKs['Android SDK'];
-    const isSDKInstalled = version !== 'Not Found';
+  win32AutomaticFix: async ({loader}) => {
+    // TODO: Create a GitHub action that checks and updates this periodically?
```

Comment:
This should probably be tracked as an issue so it can be discussed as well–and then perhaps linked to from here?

Reply:
I could, for instance, imagine this then changing to use a redirect service.